scheduler/mesos_test.go

```diff
@@ -0,0 +1,171 @@
+package scheduler
+
+import (
+	"testing"
+
+	"github.com/axsh/openvdc/model"
+	mesos "github.com/mesos/mesos-go/mesosproto"
+	"github.com/stretchr/testify/assert"
+)
+
+// to pointer
+func sP(s string) *string {
+	return &s
+}
+
+func fP(s float64) *float64 {
+	return &s
+}
+
+func iP(s int) *int32 {
+	i32 := int32(s)
+	return &i32
+}
+
+func uP(s uint64) *uint64 {
+	return &s
+}
+
+var (
+	mesosOffer *mesos.Offer
+)
+
+func init() {
+	mesosOffer = &mesos.Offer{
+		Id: &mesos.OfferID{
+			Value:            sP("d39c0128-4822-49a0-9fab-640fba518d53-O590"),
```

Comment:
``golang/protobuf`` provides conversion utilities.

```
proto.String("xxxxxx")
```

https://github.com/golang/protobuf/blob/master/proto/lib.go

Reply:
Modified to using proto library for convert to pointer.